.github/workflows/fastapi-test.yaml

```
@@ -35,15 +35,21 @@ jobs:
       - name: Install poetry
         run: |
           curl -sSL https://install.python-poetry.org | python3 - --version ${{ env.POETRY_VERSION }} -y
-          ./scripts/setup
+          ./backend/scripts/setup
       - name: black check
         run: black --diff --check $(git ls-files '*.py')
       - name: isort check
-        run: isort --diff --check --profile black $(git ls-files '*.py')
+        run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2046:warning:2:38: Quote this to prevent word splitting

(shellcheck)